keadm/app/cmd/common/types.go

```diff
@@ -250,3 +305,56 @@ type EdgeYamlSt struct {
 	EdgeHub EdgeHubSt  `yaml:"edgehub"`
 	EdgeD   EdgeDSt    `yaml:"edged"`
 }
+
+// CreateWebSocketUrl create websocket url
+func CreateWebSocketUrl(address string, port uint16, id string) string {
+	return fmt.Sprintf("wss://%s:%v/e632aba927ea4ac2b575ec1603d56f10/%s/events", address, port, id)
```

Comment:
e632aba927ea4ac2b575ec1603d56f10  --> can this be configured as well ?

Reply:
ok